lib/live_select/component.ex

```diff
@@ -514,19 +527,38 @@ defmodule LiveSelect.Component do
     })
   end
 
-  defp update_selection(nil, _current_selection, _options, _mode, _value_mapper), do: []
+  defp set_selection(nil, _current_selection, _options, _mode, _value_mapper), do: []
```

Comment:
Renamed this function to not cause confusion with the new option, as this function will effectively update the selection as a whole.

Reply:
Let us not rename it (see comment above)